node/actors/consensus/src/testonly/run.rs

```diff
@@ -154,7 +155,7 @@ async fn run_nodes(
                 for (_, recv) in network_recv {
                     s.spawn(async {
                         let mut recv = recv;
-                        use network::io;
+                        use zksync_consensus_network::io;
```

Comment:
Off-topic (?): `use` after an executed statement looks weird; it would look more natural to place it before `let`. Also, since `network` is imported into the scope, this change is not strictly necessary 🙃 (IMO, it makes sense still.)

Reply:
done